docs/google/best_practices.md

```diff
@@ -66,16 +66,16 @@ qubits = cirq.GridQubit.rect(2, 5)
 for i in range(100):
     my_circuit.append(cirq.H.on_each(*qubits))
     for q in qubits:
-        my_circuit.append(cirq.measure(q, key=f'm{q}'))
+        my_circuit.append(cirq.measure(q, key=cirq.MeasurementKey.parse_serialized(f'{i}:m{q}')))
```

Comment:
Is `parse_serialized` necessary here? I'd prefer to keep the doc examples simple if possible.

Reply:
My reason to include parse_serialized was to ensure equivalence between original circuit and shortened circuit. IIUC, in the original circuit if we repeat the same measurement key 100 times, the sampled result will contain a 3d measurements table, whereas if each measurement key is unique (which is what happens if circuit operation is expanded) then the output format will be different. 

I'll be happy to change to whatever you recommend here.

Reply:
Got it, that makes sense. Would `cirq.measure(q, key=f'{i}:m{q}')` in the original circuit work instead?

Reply:
No, we cannot specify ':' character in the key string. 
```python
: is not allowed in MeasurementKey. If this is a nested key string, use `MeasurementKey.parse_serialized` for correct behavior.
```

Reply:
Yup, that's the bit I was missing. Thanks for confirming.